warehouse/oidc/models/google.py

```diff
@@ -77,6 +77,13 @@ def __lookup_no_sub__(klass, signed_claims: SignedClaims) -> Query | None:
         __lookup_no_sub__,
     ]
 
+    @property
+    def publisher_name(self):
+        return "Google"
+
+    def publisher_url(self, claims=None):
+        return "https://accounts.google.com"
```

Comment:
For GitHub, this ends up being a link to a specific repository or SHA. I'm not sure this is useful for Google accounts. If these were always true Google accounts, we could link to `https://accounts.google.com/?authuser=user@gmail.com`, but I don't think that will work for service accounts.

Reply:
Yeah, there may not be a great URL here. Perhaps we could link to Google's OIDC docs instead?

Reply:
I think we should really only provide this if it's specific to the identity used. I think it's probably fine for this to return `None` for now, and for the templates to handle this accordingly.

Reply:
Sounds good, will update.

Reply:
Okay, updated -- I updated a handful of the uses of `publisher_url()` to prepare them for providers where it'll be `None`, but some rendering uses will require a more in-depth look with the view/form changes (e.g. the trusted publisher table on each user/project).